views/partials/header.ejs

```diff
@@ -116,14 +116,14 @@
         <h1 class='heading-large'>
             Your application will time out soon
         </h1>
-        <p>We will reset your application if you do not respond in</p>
-        <p style="margin: 0; padding: 0 10px; float: left;">
+        <p class="govuk-body">We will reset your application if you do not respond in</p>
+        <p class="govuk-body session-expiry-popup__clock">
             <img class='countdown-clock' src='/images/icon-countdown-clock.png' alt='Countdown clock'/>
-        <div style="margin-top: 1.3em;">
+        <div class="govuk-body popup__countdown">
             <span id="displayTime"></span>
         </div>
         </p>
-        <p>We do this to keep your information secure.</p>
+        <p class="govuk-body">We do this to keep your information secure.</p>
```

Comment:
sorry, I didn't spot this before, but we should use these (you'll have to convert the template to ejs). It has the right roles and aria attributes so that it can be read out nicely for someone using a screen reader

https://github.com/XGovFormBuilder/digital-form-builder/blob/9953ff825ed7e205c63374fa6906d89860517e0a/runner/public/static/modal-dialog.js

https://github.com/XGovFormBuilder/digital-form-builder/blob/9953ff825ed7e205c63374fa6906d89860517e0a/runner/src/server/views/partials/modal-dialog.html

Reply:
Maybe we should have a chat about this change tomorrow. I've had a look through the markup and I have a few questions about how some of the attributes work like `data-minutes-idle-timeout` and `data-minutes-modal-visible`.

Also, it looks like the HTML dialog element doesn't have great browser support :(
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

Maybe there's a fallback you use in the forms project or something 🤷

Reply:
They will still render since they've got styles (although you might need to dig them out? I forget) and `role="dialog"` will cover any browsers that don't support it. 

As for the attributes, they are used by the js file https://github.com/XGovFormBuilder/digital-form-builder/blob/9953ff825ed7e205c63374fa6906d89860517e0a/runner/public/static/modal-dialog.js#L25
`idleMinutesBeforeTimeOut: $('#js-modal-dialog').data('minutes-idle-timeout')`

Reply:
Yeah, I noticed the attributes in that file, haven't seen them used like that before, with data attributes but it makes sense.

Reply:
it's a jquery-ism _shudder_